hydra-web/src/js/app.js

```diff
@@ -119,17 +129,17 @@ function(
             var urlPath;
             if (tokenName == "token") {
                 urlPath = "/authentication/login";
-            } else if (tokenName == "sudo") {
-              urlPath = "/authentication/sudo";
             } else {
                 alertify.error("Unknown token name " + tokenName);
                 return;
             }
             var username = $.trim(usernameInput.value);
             var password = passwordInput.value;
+            passwordCache = passwordInput.value;
```

Comment:
Ah, i see what this does. Yeah, caching in memory isn't great. It's visible in the call stack of anything in this require block.